packages/typespec-ts/test/integration/modelInheritance.spec.ts

```diff
@@ -6,7 +6,7 @@ import TypeModelInheritanceClientFactory, {
 } from "./generated/models/inheritance/src/index.js";
 import { assert } from "chai";
 
-describe("ModelsInheritance Rest Client", () => {
+describe.skip("ModelsInheritance Rest Client", () => {
```

Comment:
Why skip?

Reply:
Because those path doesn't exist now, and you already updated the test in your custom header name PR.